addons/website_forum/models/forum.py

```diff
@@ -180,11 +180,14 @@ def _compute_count_flagged_posts(self):
     def _set_default_faq(self):
         self.faq = self.env['ir.ui.view']._render_template('website_forum.faq_accordion', {"forum": self}).decode('utf-8')
 
-    @api.model
-    def create(self, values):
-        res = super(Forum, self.with_context(mail_create_nolog=True, mail_create_nosubscribe=True)).create(values)
-        res._set_default_faq()  # will trigger a write and call update_website_count
-        return res
+    @api.model_create_multi
+    def create(self, vals_list):
+        forums = super(
+            Forum,
+            self.with_context(mail_create_nolog=True, mail_create_nosubscribe=True)
+        ).create(vals_list)
+        forums._set_default_faq()  # will trigger a write and call update_website_count
```

Comment:
Another free change, never tested ! :clap: @Feyensv 

1 commit, for 1 "free" diff, for 1 new bug.

Fixed by https://github.com/odoo/odoo/pull/76463

Reply:
It was tested. You may refer to the task and see it followed the classic process. Issue was not spotted but you cannot say "never tested".